packages/native/src/index.js

```diff
@@ -0,0 +1,28 @@
+import reactNative from 'react-native'
+import { createCss } from '@emotion/primitives-core'
+
+import { styled } from './styled'
+
+const css = createCss(reactNative.StyleSheet)
+
+const components = `ActivityIndicator ActivityIndicatorIOS ART Button DatePickerIOS DrawerLayoutAndroid
```

Comment:
Could you make this into an array since it's easier to read and there's no need to split

Reply:
Sure! No issues.